app/models/grda_warehouse/monitoring/metric_calculators/min_household_size_calculator.rb

```diff
@@ -0,0 +1,56 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module GrdaWarehouse::Monitoring::MetricCalculators
+  class MinHouseholdSizeCalculator < BaseCalculator
+    include ArelHelper
+
+    # Batch calculation for multiple clients
+    # Returns hash of { client_id => min_size }
+    def self.calculate_batch(entities, _calculation_date)
+      entity_ids = entities.map(&:id)
+
+      # First, get count of members for each [data_source_id, HouseholdID] combination
+      household_sizes = GrdaWarehouse::Hud::Enrollment.
```

Comment:
Seems like this counts exited enrollments, that's probably incorrect. Also this is the min household size over all time? 

A optimization might be to pull household_sizes out to a temporary table as it's static for every calculation (and re-used by the max-household size). Maybe we're assuming it's very cheap to build, not sure that's true

Reply:
These aren't necessarily in place to only catch things we wouldn't expect, but to alert on situations that potentially warrant investigation.  Splitting/merging households is totally fine, and unless it changes the max or min household size it wouldn't trigger an alert.  Also, it may be that some installations just leave particular alerts disabled as they aren't relevant.